common.sh

```diff
@@ -17,6 +17,8 @@ AFL_SRC=$(dirname $(dirname $SCRIPT_DIR))/AFL
 FUZZ_CXXFLAGS="-O2 -fno-omit-frame-pointer -g -fsanitize=address -fsanitize-coverage=trace-pc-guard,trace-cmp,trace-gep,trace-div"
 CORPUS=CORPUS-$EXECUTABLE_NAME_BASE
 JOBS=8
+# Every "gcloud compute ssh/scp/etc" needs --zone, otherwise throws warning
+GCLOUD_ZONE="us-west1-b"
```

Comment:
Or to make things more general, a user-supplied config file may be appropriate to define things like the zone, project name, etc.